addons/hr_holidays/models/hr_leave.py

```diff
@@ -140,6 +140,7 @@ def _default_get_request_parameters(self, values):
         'hr.employee', compute='_compute_from_holiday_type', store=True, string='Employee', index=True, readonly=False, ondelete="restrict",
         states={'cancel': [('readonly', True)], 'refuse': [('readonly', True)], 'validate1': [('readonly', True)], 'validate': [('readonly', True)]},
         tracking=True)
+    active = fields.Boolean(string="If the employee is not active, set it to false", default=True)
```

Comment:
```suggestion
    active = fields.Boolean(help="Set to false when the employee is archived", default=True)
```

The default string is enough in this case :-)